### PR TITLE
Update local provisioner docs for 3.10

### PR DESCRIPTION
**Pull request by jsafrane**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1569911

@tmorriso-rh, PTAL

### CODE REVIEW

File: install_config/configuring_local.adoc [hunk 1]
```diff
@@ -109,32 +81,28 @@ kind: ConfigMap
...
 ----
 <1> Name of the StorageClass.
 <2> Path to the directory on the host. It must be a subdirectory of *_/mnt/local-storage_*.
-<3> Path to the directory in the provisioner pod. We recommend using the same directory structure as used on the host.
+<3> Path to the directory in the provisioner pod. We recommend using the same directory structure as used on the host and `mountDir` can be omitted in this case.
 
 With this configuration, the provisioner creates:
 
 * One PV with StorageClass `local-ssd` for every subdirectory in *_/mnt/local-storage/ssd_*.
 * One PV with StorageClass `local-hdd` for every subdirectory in *_/mnt/local-storage/hdd_*.
```
Comment (qinpingli) [inline install_config/configuring_local.adoc]: In openshift 3.10, local provisioner will create PV just for mountpoint.

**Comment by qinpingli** [comment]
In openshift 3.10, we can remove the "Adding New Devices" part.

**Comment by qinpingli** [comment]
Do we need add a part for how to using local volume from WEBUI in this PR?

File: install_config/configuring_local.adoc (hunk 1 above)
Comment (jsafrane) [reply install_config/configuring_local.adoc]: Fixed

**Comment by jsafrane** [comment]
> In openshift 3.10, we can remove the "Adding New Devices" part or change this part.


I kept it there to mention SELinux relabeling.

**Comment by jsafrane** [comment]
TODO: add documentation about block devices

**Comment by jsafrane** [comment]
> TODO: add documentation about block devices

block devices are alpha upstream, so no documentation nor support

File: install_config/configuring_local.adoc [hunk 2]
```diff
@@ -27,37 +27,9 @@ The local volume provisioner allows using local storage within {product-title} a
 
 [NOTE]
 ====
-Local volumes is an alpha feature and may change in a future release of {product-title}.
+Local volumes is a technical preview feature and may change in a future release of {product-title}.
```
Comment (tmorriso-rh) [inline install_config/configuring_local.adoc]: s/technical preview/Technology Preview 

You should also add the IMPORTANT note: https://mojo.redhat.com/docs/DOC-1124368

File: install_config/configuring_local.adoc [hunk 3]
```diff
@@ -109,32 +81,28 @@ kind: ConfigMap
...
 
-* One PV with StorageClass `local-ssd` for every subdirectory in *_/mnt/local-storage/ssd_*.
-* One PV with StorageClass `local-hdd` for every subdirectory in *_/mnt/local-storage/hdd_*.
+* One PV with StorageClass `local-ssd` for every mounted subdirectory in *_/mnt/local-storage/ssd_*.
+* One PV with StorageClass `local-hdd` for every mounted subdirectory in *_/mnt/local-storage/hdd_*.
 
-The `LocalPersistentVolumes` alpha feature now also requires the `VolumeScheduling` alpha feature. This is a breaking change, and the following changes are required:
+[WARNING]
+====
+Syntax of the ConfigMap has changed between {product-title} 3.9 and 3.10. Since this feature is in technical preview, the ConfigMap is not converted automatically during update.
```
Comment (tmorriso-rh) [inline install_config/configuring_local.adoc]: s/technical preview/Technology Preview

File: install_config/configuring_local.adoc [hunk 4]
```diff
@@ -189,13 +157,14 @@ The provisioner runs as root to be able to clean up the directories when a PV is
 
 [[local-volume-adding-new-devices]]
 === Adding New Devices
-Adding a new device requires several manual steps:
+Adding new device is semi-automatic. The provisioner periodically checks for new mounts in configured directories. Admin just needs to create a new subdirectory there, mount a device there and allow pods to use the device by applying SELinux label on it:
```
Comment (tmorriso-rh) [inline install_config/configuring_local.adoc]: s/Adding new device/Adding a new device
s/Admin just needs to/The administrator needs to 
s/mount a device there/mount a device there,
s/by applying SELinux label on it:/by applying the SELinux label:

**Comment by tmorriso-rh** [comment]
@jsafrane I suggested a few edits. Thanks.

File: install_config/configuring_local.adoc (hunk 2 above)
Comment (jsafrane) [reply install_config/configuring_local.adoc]: Replaced with IMPORTANT from the mojo page.

File: install_config/configuring_local.adoc (hunk 3 above)
Comment (jsafrane) [reply install_config/configuring_local.adoc]: Fixed

File: install_config/configuring_local.adoc (hunk 4 above)
Comment (jsafrane) [reply install_config/configuring_local.adoc]: Fixed

**Comment by jsafrane** [comment]
@tmorriso-rh, I applied you suggestions, PTAL

**Comment by tmorriso-rh** [comment]
@jsafrane I think this is ready to merge. Can you remove/squash the extra commit? We require all PRs to have only 1 commit before we can merge. By squashing your merge commits into one commit, you are making it easier for us to understand the scope of your work. Thanks.

**Pull request merged**